src/components/BulkEdit.test.js

```diff
@@ -10,6 +10,7 @@ import { runAxeTest } from '@folio/stripes-testing';
 
 import '../../test/jest/__mock__';
 
+import { IntlProvider } from 'react-intl';
```

Comment:
Please move the import higher to be with other third party imports

Reply:
done!